connect/client/logger.py

```diff
@@ -6,6 +6,14 @@ class RequestLogger:
     def __init__(self, file=sys.stdout):
         self._file = file
 
+    def obfuscate(self, key, value):
```

Comment:
IMO it is better to call this method obfuscate_authorization and pass just the value as the only argument.